.github/workflows/ci.yml

```diff
@@ -13,15 +13,21 @@ env:
   UGBIO_BASE_IMAGE_NAME: ugbio_base:1.0
 
 jobs:
+  get-workspaces:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{steps.list_dirs.outputs.matrix}}
+    steps:
+      - uses: actions/checkout@v4
+      - id: list_dirs
+        run: echo "::set-output name=matrix::$(ls src   | grep -v "__" | grep -v "/\." | sed 's|src/||' | jq -R -s -c 'split("\n")[:-1]')"
```

Comment:
something simpler like "ls -d */" didn't work?

Reply:
there can be folder like __pycache__ that we want to filter out